sdk/deviceupdate/azure-iot-deviceupdate/swagger/README.md

```diff
@@ -20,7 +20,7 @@ autorest --java --use=C:/work/autorest.java
 
 ## Generate autorest code
 ```yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/b9b91929c304f8fb44002267b6c98d9fb9dde014/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2022-07-01-preview/deviceupdate.json
+input-file: https://raw.githubusercontent.com/dpokluda/azure-rest-api-specs/bbf13b0366e5863ce65f84d588ed5681326af0cb/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json
```

Comment:
I think you need to switch to Azure main branch (maybe another regen), before PR can be merged.

Reply:
If you have swagger PR pending, let us know the link and state of it (whether it is approved, or there is change request on it that likely would result in SDK change).

Reply:
Updated.